src/Components/SmartComponents/Systems/Systems.js

```diff
@@ -114,6 +123,37 @@ class Systems extends React.Component {
         this.apply({ ...newFilter, page: 1 });
     };
 
+    doOptOut = () => {
+        if (this.state.selected.size > 0) {
+            this.props.optOutSystems(this.state.selected, !this.state.opt_out).then(() => {
+                const params = omit(this.state,
```

Comment:
Can we move this to separate method as it repeats multiple times on this page? (Mainly because of listing parameters as string resulting in potential error in the future).